src/tokenizer/tokenizer.py

```diff
@@ -648,6 +654,13 @@ def parse_digits(w):
         sec = int(p[2])
         if (0 <= h < 24) and (0 <= m < 60) and (0 <= sec < 60):
             return TOK.Time(w, h, m, sec), s.end()
+    s = re.match(r'\d+[A-Za-z]{1}', w)
```

Comment:
I suggest moving this to after the time and date checks (HH:MM and YYYY/MM/DD and derivatives), considering those as more specific and this as slightly less specific. Also, maybe the regex should end with something like `(?![:alpha:])` to prevent "10million" or "1000isk" from matching. We only want a number plus one character, not multiple chars.